### PR TITLE
[Fix](exectuor)Fix Follower Fe query queue may not work when exec alter

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
When exec Alter query queue stmt in master FE, the alter stmt can be replayed in follower FE.
The follower FE replayed workload group meta and create a new QueryQueue, this means there maybe multipe QueryQueue at the same time in follower FE.
This could cause follower FE can not limit query as Workload group's property.

## Solution
Move QueryQueue from WorkloadGroup to WorkloadGroupManager.
WorkloadGroup only keeps queryQueue's property.
But QueryQueue's lifecycle is managed by WorkloadGroupManager.

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/QueryQueue.java [hunk 1]
```diff
@@ -44,6 +47,12 @@ public class QueryQueue {
     public static final String RUNNING_QUERY_NUM = "running_query_num";
     public static final String WAITING_QUERY_NUM = "waiting_query_num";
 
+    private Thread updatePropThread;
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/QueryQueue.java:50]: 我们写一个thread
1. 遍历所有的workloadgroup
2. 检查每个workload group 里的query queue的配置和workloadgroup 元数据对比，如果不同，那么就reset queue 的设置。

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java [hunk 2]
```diff
@@ -75,9 +76,65 @@ public class WorkloadGroupMgr implements Writable, GsonPostProcessable {
...
+
+    public void startUpdateThread() {
+        WorkloadGroupMgr wgMgr = this;
+        updatePropThread = new Thread(new Runnable() {
+            public void run() {
+                while (!wgMgr.stopped) {
+                    try {
+                        wgMgr.resetQueryQueueProp();
+                        Thread.sleep(Config.query_queue_update_interval_ms);
+                    } catch (Exception e) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java:98]: catch Throwable

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java [hunk 3]
```diff
@@ -75,9 +76,65 @@ public class WorkloadGroupMgr implements Writable, GsonPostProcessable {
...
     private final Map<Long, WorkloadGroup> idToWorkloadGroup = Maps.newHashMap();
     private final Map<String, WorkloadGroup> nameToWorkloadGroup = Maps.newHashMap();
+    private final Map<Long, QueryQueue> idToQueryQueue = Maps.newHashMap();
     private final ResourceProcNode procNode = new ResourceProcNode();
     private final ReentrantReadWriteLock lock = new ReentrantReadWriteLock();
 
+    private Thread updatePropThread;
+    private volatile boolean stopped = false;
+
+    public void stopUpdateThread() {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java:86]: stop never called？

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**